cypress/integration/tests/controls/stakeholdergroups/interlinked.test.ts

```diff
@@ -0,0 +1,96 @@
+/// <reference types="cypress" />
+
+import {
+    login,
+    selectItemsPerPage,
+    click,
+    clickByText,
+    exists,
+    notExists,
+} from "../../../../utils/utils";
+import { navTab } from "../../../views/menu.view";
+import { Stakeholdergroups } from "../../../models/stakeholdergroups";
+import { Stakeholders } from "../../../models/stakeholders";
+import { tdTag, trTag, stakeholdergroups } from "../../../types/constants";
+import { expandRow } from "../../../views/commoncontrols.view";
+import * as data from "../../../../utils/data_utils";
+
+describe("Single Stakeholder group linked to a stakeholder member", () => {
+    beforeEach("Login", function () {
+        // Perform login
+        login();
+
+        // Interceptors for stakeholder groups
+        cy.intercept("POST", "/api/controls/stakeholder-group*").as("postStakeholdergroups");
+        cy.intercept("GET", "/api/controls/stakeholder-group*").as("getStakeholdergroups");
+
+        // Interceptors for stakeholders
+        cy.intercept("POST", "/api/controls/stakeholder*").as("postStakeholder");
+        cy.intercept("GET", "/api/controls/stakeholder*").as("getStakeholders");
+    });
+
+    it("Attach and edit stakeholder attached to Stakeholder group", function () {
```

Comment:
We can name it : 
stakeholder attach, update and delete dependency on stakeholdergroup